capa/main.py

```diff
@@ -637,7 +637,7 @@ def get_rules(
 
     total_rule_count = len(rule_file_paths)
     for i, (path, content) in enumerate(zip(rule_file_paths, rule_contents)):
-        on_load_rule(path, i, total_rule_count)
+        on_load_rule(path, i + 1, total_rule_count)
```

Comment:
`1` to `n` instead of `0` to `n-1`

Reply:
what do you think about changing the rendering of `i` within the callbacks, rather than changing the index here? `i` is an index in the code, which is 0-based, so as a programmer i'd expect this to range (0, n-1). 

its true that the change as-is touches fewer lines, but i think its more surprising.

Reply:
sounds reasonable; let's move it to the rendering functions.